url.py

```diff
@@ -0,0 +1,63 @@
+import hashlib
+
+class URL:
+    # key: long_url, value: URL()
+    URL_DICT = {}
+
+    def __init__(self, long_url: str) -> None:
+        self.long_url = long_url
+        self.short_url = "short_url"
+        self.access_count = 0
+
+    def shorten_url(long_url):
+        if long_url.startswith('www.') is False:
+            raise ValueError
```

Comment:
I like the idea of error checking here, but I think you'd want to call out in an interview this is more a proof of concept than 100% foolproof. Not all URLs start with `www`.

Reply:
That was my thinking. I'm sure there's a module out there I could use to validate URL's more effectively.